### DISCUSSION TITLE
How do I get frigate to send the image to Double Take on Demand

### DISCUSSION THREAD

**Discussion by mmitch2k** [blakeblackshear/frigate#5421] [Ask A Question]
How do i create an automation to have a camera ‘re-scan’ the room. Im using Doubletake, Frigate, Compreface. Ideally i would like to run this automation when my motion sensors detect motion.  Frigate seems slow to respond to motion and sometimes does not respond at all.

Thanks

**Comment by NickM-27** [comment]
> Frigate seems slow to respond to motion and sometimes does not respond at all.

this is likely a config error

do you have doubletake configured with

```
frigate:
  update_sub_labels: true
```

which will get doubletake to send face matches back to frigate

**Reply by NickM-27** [reply]
doubletake will by default send a snapshot every time it sees a face until it gets a positive match

**Reply by mmitch2k** [reply]
So here is the use case. I have 5 cameras from Room 1 to Room 5. If i walk across each room sometimes it wont send a snapshot until I get to room 5. Or sometimes I may be in Room 1 and I need it to confirm Im still in room 1, so I was thinking it would be great if I can run an automation every 5 minutes to have it send me a snapshot to confirm im still there. Or sometimes double take mis-labels me and puts me in room4 when im actually in room 5. I think an automated 're-scan' would help in these situations. 

Hope this makes sense

**Reply by mmitch2k** [reply]
> > Frigate seems slow to respond to motion and sometimes does not respond at all.
> 
> this is likely a config error
> 
> do you have doubletake configured with
> 
> ```
> frigate:
>   update_sub_labels: true
> ```
> 
> which will get doubletake to send face matches back to frigate

My double take config only has the ip for frigate

frigate:
  url: http://xx.xx.xx.xx:xxxx

**Reply by NickM-27** [reply]
Would definitely recommend enabling the update sub labels setting.

**Reply by NickM-27** [reply]
Also, doubletake has a manual upload API, you'd need to use that.

**Reply by mmitch2k** [reply]
OK i have enabled the sub labels. And will check out the api documentation.

Thank you for your help.
